lib/ansiblelint/cli.py

```diff
@@ -194,6 +196,9 @@ def merge_config(file_config, cli_config) -> NamedTuple:
     if 'skip_list' in file_config:
         cli_config.skip_list = cli_config.skip_list + file_config['skip_list']
 
+    if 'warn_list' in file_config:
+        cli_config.warn_list = cli_config.skip_list + file_config['warn_list']
```

Comment:
I think this is incorrect - cli_config.skip_list is not what you are working with - I think it should be cli_config.warn_list

Reply:
Indeed. Thanks, that was a genuine bug.